packages/yoga/src/Progress/web/Progress.jsx

```diff
@@ -59,14 +60,16 @@ const ProgressBar = styled.progress`
 `}
 `;
 
-const Label = styled.label`
+const Label = styled(Text.Tiny)`
```

Comment:
The `Text.Tiny` has the `p` element. It would be nice to keep the `label`

Reply:
Ok :D